support-frontend/assets/pages/[countryGroupId]/checkout/components/PersonalDetailsFields.tsx

```diff
@@ -45,6 +55,7 @@ export function PersonalDetailsFields({
 					}}
 					onBlur={(event) => {
 						event.target.checkValidity();
+						confirmEmailRef.current?.querySelector('input')?.checkValidity();
```

Comment:
I assume the UX would be that as soon as a user tabs out of the email field we'll invalidate the confirm email field assuming the values don't match at this point, which is most likely as they complete email first before confirm email? Would it be better UX to only check the validity of the input if it has a `.value.length > 0` ?